modules/social_features/social_search/config/install/search_api.index.social_all.yml

```diff
@@ -138,8 +136,10 @@ field_settings:
       view_mode:
         'entity:group':
           closed_group: teaser
+          flexible_group: teaser
```

Comment:
flexible/secret are optional modules, most likely it's enabled in your environment but we shouldn't have it in here as they might be disabled for the users.